### PR TITLE
Add debug configuration for VSCode

### PR DESCRIPTION
**Pull request by jthistle**
This allows automatic debugging from within the VSCode editor. I've also included a small change to the other config file to include the telemetry directory.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+      // Use IntelliSense to learn about possible attributes.
+      // Hover to view descriptions of existing attributes.
+      // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (dmitrio95) [inline .vscode/launch.json]: Do JSON files support comments like this one?

Comment (Howard-C) [reply .vscode/launch.json]: Yes, despite that weird highlighting on GitHub

Comment (jthistle) [reply .vscode/launch.json]: Yeah, VSCode uses JSON 2.0, which supports comments but had the same file extension as JSON 1.0.

Comment (dmitrio95) [reply .vscode/launch.json]: Ah, OK, thanks for explanation!

**Pull request merged**